python-squid/src/main/java/org/sonar/python/tree/PythonTreeMaker.java

```diff
@@ -518,9 +521,62 @@ private PyExceptClauseTree exceptClause(AstNode except, PyStatementListTree body
   }
 
   PyExpressionTree expression(AstNode astNode) {
+    if (astNode.is(PythonGrammar.TEST) && astNode.getChildren().size() == 1) {
+      return expression(astNode.getFirstChild());
```

Comment:
I think we should return `expression(astNode.getFirstChild())` in all cases where `astNode.getChildren().size() == 1`.